src/routes/+layout.svelte

```diff
@@ -52,8 +57,13 @@
     height: auto;
     width: 100%;
   }
+  .gradient {
+    background-image: linear-gradient(#f0a9a8 75%, #9e9e9f00 100%);
+  }
   .logo-desktop {
     width: 100%;
+    z-index: 10;
+    padding-bottom: 30px;
```

Comment:
why?

Reply:
It is so that the gradient does not start as high as the logo does not have a margin, so trimming it with this padding makes the fade more natural

Reply:
Ok, then put it into a var with a good name so this is more clear --fade-outset for example